cppcache/include/geode/internal/chrono/duration.hpp

```diff
@@ -57,6 +57,12 @@ template <>
 struct _suffix<std::micro> {
   static constexpr char const* value = "us";
 };
+#ifdef WIN32
```

Comment:
No! Let's please leave ticks out and stick with SI units. Please for the love all all things holy!